src/DotNet.Status.Web/Controllers/GitHubHookController.cs

```diff
@@ -49,16 +51,26 @@ private async Task EnsureLabelsAsync()
         }
 
         [GitHubWebHook(EventName = "issues")]
-        public async Task<IActionResult> IssuesHook(IssuesHookData data)
+        public async Task<IActionResult> IssuesHook(JsonElement data)
```

Comment:
Can't this be a string to avoid double deserialization?

Reply:
nope, tested failing for, request header 'Content-Type =  application/json` so ASP .net expect json and not string